scripts/run_ci_tasks.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#####################################################
+# This script is used for Continuous Integration
+#
+# Run locally to verify before committing your code.
+#
+# Options:
+#   -z to run Flutter Analysis
+#   -a to run Android CI tasks.
+#   -i to run iOS CI tasks.
+#####################################################
+
+set -o pipefail
+set -e
+
+# get platforms
+ANDROID=true
+ANALYZE=true
+IOS=true
+
+# Parse arguments
+OPTS=`getopt haiz $*`
+if [ $? != 0 ] ; then echo "Failed parsing options." >&2 ; exit 1 ; fi
+eval set -- "$OPTS"
+
+while true; do
```

Comment:
Can we remove this optional command for now?

Reply:
going to keep it because it makes it easier to troubleshoot locally